src/command-line/index.js

```diff
@@ -12,15 +13,21 @@ program.option("    --home <path>" , "home path");
 
 var argv = program.parseOptions(process.argv);
 if (program.home) {
-	Helper.HOME = program.home;
+	Helper.HOME = path.resolve(program.home);
 }
 
-var config = Helper.HOME + "/config.js";
+var config = Helper.CONFIG_PATH;
 if (!fs.existsSync(config)) {
 	mkdirp.sync(Helper.HOME, {mode: "0700"});
 	fs.writeFileSync(
 		config,
-		fs.readFileSync(__dirname + "/../../defaults/config.js")
+		fs.readFileSync(path.resolve(path.join(
+			__dirname,
```

Comment:
[This](https://github.com/thelounge/lounge/blob/c8df0d68c02f30a9cb0c7c7fa93d93479654223b/src/plugins/irc-events/ctcp.js#L1) uses something else, I'm thinking the goal is the same. Should we stick with one or the other?